lms/models/assignment.py

```diff
@@ -86,6 +86,8 @@ class Assignment(CreatedUpdatedMixin, Base):
         "AssignmentMembership", lazy="dynamic", viewonly=True
     )
 
+    course_id: Mapped[int | None] = mapped_column(sa.ForeignKey(Course.id))
```

Comment:
This model now has both a `course_id` field and a `course` property which get results in different ways. Will the `course` property be removed in future?

Reply:
Yes, following the steps of the PoC PR: 
https://github.com/hypothesis/lms/pull/6517/commits/c18f986effd2725a9abac99b09fe89c7396bd993#diff-71bbc86f39f14ca26fa8c345791473dcc66487d919a863426add9f9a24d6e915L95

That would be the last step of this train of PRs :+1: